requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx
```

Comment:
can we move this to `docs`? it looks like the github action has a parameter that can help us specify the location of the requirements.txt